rgw-standalone.yml will also now collect keys

andrewschoen opened the pull request:
This playbook is used to add an rgw node to an existing cluster when you
don't want to rerun the site.yml against your entire cluster again.

This is also used by the api/rgw/configure endpoint of ceph-installer.

Signed-off-by: Andrew Schoen <aschoen@redhat.com>

leseb commented:
We already have https://github.com/ceph/ceph-ansible/blob/master/infrastructure-playbooks/rgw-standalone.yml

andrewschoen commented:
@leseb I don't believe ``rgw-standalone.yml`` will work in the ceph-installer use case because we also need to fetch keys and can't depend on what's in the ``fetch_directory`` to be correct.

andrewschoen commented:
@leseb I've modified this so that ``rgw-standalone.yml`` also collects keys so that I can use it with ceph-installer and removed ``rgw-configure.yml``
